src/olympia/zadmin/tasks.py

```diff
@@ -587,14 +587,30 @@ def fetch_langpack(url, xpi, **kw):
             log.info('[@None] Created new "{0}" language pack, version {1}'
                      .format(xpi, data['version']))
 
+        # Set the category
+        for app in version.compatible_apps:
+            static_category = (
+                CATEGORIES[app.id][amo.ADDON_LPAPP]['general'])
```

Comment:
Just making sure… Android doesn't have any `ADDON_LPAPP` key defined, so if this is a real scenario (which I don't know) we might want to add a check here.